src/OpenMagneticsVirtualBuilder/builder.py

```diff
@@ -35,13 +35,13 @@
     sys.path.append(f"{freecad_path}\\Mod\\Arch")
 else:
     sys.path.insert(0, "/usr/lib/python3/dist-packages")
-    sys.path.append("/usr/lib/freecad-daily/lib")
-    sys.path.append("/usr/share/freecad-daily/Ext")
-    sys.path.append("/usr/share/freecad-daily/Mod")
-    sys.path.append("/usr/share/freecad-daily/Mod/Part")
-    sys.path.append("/usr/share/freecad-daily/Mod/Draft")
-    sys.path.append("/usr/share/freecad-daily/Mod/Draft/draftobjects")
-    # Comment line 31 from /usr/share/freecad-daily/Mod/Draft/draftutils/params.py if it crashes at import
+    sys.path.append("/usr/local/lib")
+    sys.path.append("/usr/local/Ext")
+    sys.path.append("/usr/local/Mod")
+    sys.path.append("/usr/local/Mod/Part")
+    sys.path.append("/usr/local/Mod/Draft")
+    sys.path.append("/usr/local/Mod/Draft/draftobjects")
+    # Comment line 31 from /usr/local/Mod/Draft/draftutils/params.py if it crashes at import
```

Comment:
I have added a logic to check for paths in different folders which should cover all cases not. Please merge with main branch

Reply:
added the logic to mine.